webJsInterfaceSupported.json

```diff
@@ -1,4 +1,4 @@
 {
     "_comment": "contains the web js interface version that this package supports",
```

Comment:
size limit and pre commit pr check are failing

Reply:
They failed because they need the new website version to build (with `getAccessToken`)